### PR TITLE
Release 2.8.0

### PR DESCRIPTION
**Pull request by Climbatize**
merge release candidate into master

### CODE REVIEW

**Review by ts-leojoseph-r — COMMENTED** [review]
I see lot of changes related to refactoring and location. Can we avoid this in Release PR's ?

**Comment by Climbatize** [comment]
> I see lot of changes related to refactoring and location. Can we avoid this in Release PR's ?

no worries, these are the fixes that resulted from regression tests. In fact this is just the merge of the released v2.8.0 SDK ;)

File: scripts/deploy-to-app-center.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```
Comment (corycaywood) [inline scripts/deploy-to-app-center.sh:1]: I think this isn't correct - `#!/bin/sh` is used for system boot scripts, isn't it?

When you create a new script in the Bitrise UI, it automatically uses `#!/usr/bin/env bash` for Bash scripts, so I think we should keep that.

Comment (Climbatize) [reply scripts/deploy-to-app-center.sh]: you're right, this was a failed attempt to fix something wrong

**Review by corycaywood — APPROVED** [review]
👍

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**